### ISSUE TITLE
Deft Explorer not working

### ISSUE THREAD

**Issue by JediMastaDJ** [Metacomet10/MPMB-v13-content-catchup#26]
Deft Explorer only works if the UA ranger is active. Looks like the script is disabled at this point for use without the UA. I haven't tested if activating this script section will break the sheet yet. Will try tonight to verify.

https://github.com/Metacomet10/MPMB-v13-content-catchup/blob/3e5795aa114337377a9dc01c7b8dbd3c14887d29/Content/pub_20201117_TCoE-WIP_3P.js#L5181

**Comment by Metacomet10** [comment]
Unfortunately, this is intended. The Ranger OCF code is very broken. The official release of Tasha's content by MPMB is expected soon (TM), and so instead of spending lots of time and effort to fix code that will be obsolete very quickly, the problematic code was commented out.

I'm going to leave this issue open so others see it.

**Comment by JediMastaDJ** [comment]
Thanks. I realized that after I posted this. I've looked at it for the past few days and played around with it. 

I think there's a conflict due to identical coding between the UA:CFV in the main source and the one in the TCoE. I played around with it but I wasn't able to find a reliable way to quiry the active source list to check if the UA:CFV was enabled. It would only return that the source was present and always skip the code (which commenting did anyways).

I've enabled the code for now on my sheet knowing that I won't be using the UA content, since it only breaks when the UA:CFV is enabled.

I didn't realize he was that close to the new content. Thanks for that information. I'll just end up recreating it when he releases the new batch.

Thanks for your time.
